src/services/helpers/getErrorMessage.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2018-2020 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { AxiosResponse } from 'axios';
+
+export function getErrorMessage(error: Error | AxiosResponse): string {
```

Comment:
it does not seem to work in all cases yet.
![Screenshot_20210623_160349](https://user-images.githubusercontent.com/5887312/123103620-df0a7f00-d43e-11eb-8b67-11e8dbf90411.png)

Reply:
Now should be fixed 
![Screenshot_20210623_165125](https://user-images.githubusercontent.com/5887312/123108773-60fca700-d443-11eb-9e33-3f62b95f0ef7.png)